include/openssl/evp.h

```diff
@@ -186,7 +189,7 @@ int (*EVP_MD_meth_get_ctrl(const EVP_MD *md))(EVP_MD_CTX *ctx, int cmd,
  * if the following flag is set.
  */
 # define EVP_MD_CTX_FLAG_FINALISE        0x0200
-/* NOTE: 0x0400 is reserved for internal usage in evp_int.h */
+/* NOTE: 0x0400 is reserved for internal usage in "crypto/evp.h" */
```

Comment:
We shouldn't mention internal headers in public headers

Reply:
So just mention that it's reserved, right?

```C
/* NOTE: 0x0400 is reserved for internal usage */
```